css/style.css

```diff
@@ -0,0 +1,124 @@
+
+body {
+  background-color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  padding: 0;
+  margin: 0;
+}
+
+/* HEADER-MOBILE */
+header {
+  padding: 0;
+  margin: 1px;
+  display: flex;
+  width: 100;
+  height: 1fr;
+  background-color: rgb(255, 255, 255);
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+}
+
+#main-container .introduction h1 {
+  font-size: 40px;
+  text-align: center;
+  color: #172b4b;
+  font-weight: 700;
+
+  /* padding-top: 70px ; */
+  margin: 70px 40px 10px 40px;
+}
+
+header a {
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  text-decoration: none;
+  font-size: 20px;
+  margin-left: 35px;
+  margin-top: 10px;
+  color: rgba(0, 0, 0, 0.774);
+  font-weight: 700;
+}
+
+header #bmenu {
+  border: none;
+  background-color: transparent;
+  cursor: pointer;
+}
+
+header #bmenu .material-symbols-outlined {
+  margin: 20px 35px 0 0;
+  font-size: 35px;
+  color: black;
+}
+
+/* INTRODUCCTION-SECTION */
+#main-container .introduction {
+  background-image: url(../img/background-mobile.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  display: inline-block;
+}
+
+#main-container .introduction h2 {
+  font-size: 40px;
+  text-align: center;
+  color: #36b37e;
+  font-weight: 700;
+  margin: 10px 45px 56px 45px;
+}
```

Comment:
- [ ] As suggested by the previous reviewer [here](https://github.com/SamuelTTW/Portfolio-Samuel-Parra-2/pull/1#discussion_r884678722). kindly add the font family "Inter" for introduction h2.